Solutions/Corvus.Monitoring.Instrumentation.Abstractions/Corvus/Monitoring/Instrumentation/NullOperationsInstrumentation.cs

```diff
@@ -26,8 +26,14 @@ private class Operation : IOperationInstance
         {
             public static readonly Operation Instance = new();
 
-            public void AddOperationDetail(AdditionalInstrumentationDetail detail)
+            public void AddOperationMetric(string name, double value)
             {
+                throw new System.NotImplementedException();
```

Comment:
Why the exception? The purpose of the `NullOperationsInstrumentation` is to act as a bit bucket in cases where an application has ended up using a framework that's going to use `IOperationsInstrumentation`, but where for whatever reason, the app just doesn't want to collect any telemetry (e.g., it's a simple demo app).

So this is meant to be a "do nothing" type, not a "blow up if someone tries to use this" type.